src/accountlist/accountlistReducers.js

```diff
@@ -0,0 +1,57 @@
+import { fromJS } from 'immutable';
+
+import {
+  REALM_ADD,
+  LOGIN_SUCCESS,
+  LOGOUT,
+  ACCOUNT_REMOVE,
+} from '../account/accountActions';
+
+const initialState = fromJS([]);
+
+export default (state = initialState, action) => {
```

Comment:
I don't think I like the pattern of having the first account in the list be the active account. It's strange to have to remove an account and add it back to the front of the list

Why not store the list of accounts as a map and have `activeAccountId` as part of the state (like it used to be before the auth refactor)?